css/styles.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: red;
+}
```

Comment:
- [ ] Kindly make sure your `style.css` file includes one class with styles, instead of `h1`, for your header as required for the project i.e you should add a `class` in the `<h1>` tag (in the `index.html` file) and replace it(class_name you added) with `h1` tag in the `style.css` file for styling. I know you can do that :muscle: :muscle: